.markdownlint.jsonc

```diff
@@ -0,0 +1,6 @@
+// For information on writing markdownlint configuration see:
+//  https://github.com/DavidAnson/markdownlint/blob/main/README.md#optionsconfig
+{
+    "MD013": false,  // Execism tends to break lines at sentence ends rather than at a column limit
+    "MD024": false   // The format for instructions.md requires  repeated headings, e.g. "Task"
+}
```

Comment:
I'm kinda puzzled here, this file (and the others) has a trailing newline on my system.
My git config does not have any settings for new lines, trailing or otherwise.
VSCode shows newlines and does not have any settings that should remove them, and is set to use LF.
Yet if I open in VIM, it is quite apparent that the newlines are missing :(

Reply:
With my latest commit, I have removed the newline I see in my editor and pushed. GitHub now also reports no trailing newline.
I think this inidicates the problem is on your side Victor 😨

Reply:
It is on my side, but also mirrors as reported on github, as I
verified it in the patch on the web in Github's site (see below).

I would not remove the newline that belongs at the last line of the text
file, but ensure that there is a new line for the last line of the file.

You can see that the newline is missing (or not) by
[adding `.patch` to the https://github.com/exercism/problem-specifications/pull/1895 URL][1]
and then searching for the phrase "\\ No newline at end of file" like it
is shown when reported in the local `git log --patch -n1` as if the
current head were to have a missing newline at the end of the file, or
if it were corrected from the prior commit.

[1]: https://github.com/exercism/problem-specifications/pull/1895.patch

Reply:
It is on my side though, because I did not look down the patch list far enough.  (And because I did not apply the patch to verify locally, I would have noticed it right away, so me being lazy. )

Reply:
Well this is very odd.
I can definitely see that the patch diff that GitHub creates is missing the new line for this file.
However in the web interface I also see that there is a newline.

Reply:
Ah. When you add `.patch` at the end of the URL, it shows patches for all the commits separately - including the earlier commits that did indeed have the newline missing.

If I scroll to the bottom of that patch diff page and then scroll up a little bit, we can see:

```text
From 898d2c9b09f4a23aec023f6145f172d636cac17f Mon Sep 17 00:00:00 2001
From: wolf99 <281700+wolf99@users.noreply.github.com>
Date: Mon, 10 Jan 2022 20:53:26 +0000
Subject: [PATCH 08/11] Revert "test no trailing newline"

This reverts commit b97d854b27d636ee0929a38017c6b17c7f4b69d9.
---
 .markdownlint.jsonc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/.markdownlint.jsonc b/.markdownlint.jsonc
index 9f24e90ac..1790e7032 100644
--- a/.markdownlint.jsonc
+++ b/.markdownlint.jsonc
@@ -1,6 +1,6 @@
 // For information on writing markdownlint configuration see:
 //  https://github.com/DavidAnson/markdownlint/blob/main/README.md#optionsconfig
 {
-    "MD013": false,  // Exercism tends to break lines at sentence ends rather than at a column limit
+    "MD013": false,  // Execism tends to break lines at sentence ends rather than at a column limit
     "MD024": false   // The format for instructions.md requires  repeated headings, e.g. "Task"
-}
\ No newline at end of file
+}
```

Reply:
Why it thinks it is the 17th of  September 2001 I have no idea! 😆

Reply:
Yep, so you now see how my laziness caused me to see what I saw without going further down to see that it was corrected in a later patch.